Sources/libhostmgr/Platforms/VMManager.swift

```diff
@@ -14,6 +14,11 @@ public struct VMManager {
         self.console = console ?? Console()
     }
 
+    enum Errors: Error {
+        // trying to establish an SSH connection timed out
+        case sshConnectionTimeout
```

Comment:
Nitpick: This way, we can this useful comment as documentation and in the quick help tooltip.

```suggestion
        /// Trying to establish an SSH connection timed out
        case sshConnectionTimeout
```

Reply:
Good catch. 👍